src/providers/api/api.ts

```diff
@@ -0,0 +1,25 @@
+export class ApiProvider {
+  isProduction = true;
+  private config = {
+    prod: {
+      bitcore: 'https://ducws.rocknblock.io',
+      ducatuscoins: 'https://www.ducatuscoins.com',
+      crowdsale: 'https://tokenization.centuriongm.com',
+      pog: 'https://d-pog.com'
```

Comment:
Надо добавить еще один параметр для API
```suggestion
     pog: 'https://d-pog.com',
     ratesApi: 'https://rates.ducatuscoins.com/api/v1/rates/'
```